src/tools/teardown-cf-stack.ts

```diff
@@ -149,20 +149,44 @@ const destroyStack = async (stackName: string): Promise<StackResource[]> => {
         process.exit(-1);
       }
     }
-    if (result.StackResources === undefined) {
+    if (result.StackResourceSummaries === undefined) {
       console.log("Got empty response from AWS, aborting...");
       process.exit(-1);
     }
-    const numberDeleted = result.StackResources.filter(
+
+    resources.push(...result.StackResourceSummaries);
+    nextToken = result.NextToken;
+  } while (nextToken);
+  return resources;
+};
+
+const destroyStack = async (
+  stackName: string
+): Promise<StackResourceSummary[]> => {
+  const deleteResult = await cfClient.send(
+    new DeleteStackCommand({ StackName: stackName })
+  );
+  if (isAWSError(deleteResult)) {
+    console.log(deleteResult.error.toString());
+    process.exit(-1);
+  }
+  let resourcesFailed: StackResourceSummary[] = [];
+  while (true) {
```

Comment:
For anyone else who gets spooked seeing this here (or even to myself if I circle back to this) this look is broken by invoking `process.exit`